gigadb/app/configurator/src/dotfiles.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+# source .env if existing, otherwise create a new from default example then source it
+function makeDotEnv () {
+  mdeBaseDir=$1
+
+  if [ -f  "$mdeBaseDir/.env" ];then
+      echo "An .env file is present"
+  elif [ -f  "$mdeBaseDir/config-sources/env.example" ];then
+    echo "An .env file wasn't present, creating a new one from the default example"
+    cp "$mdeBaseDir/config-sources/env.example" "$mdeBaseDir/.env"
+  else
+      echo "Neither .env file or default example were present, generating one on the fly"
+      currentEnv=dev
+      username=$(git config --get user.name | cut -d" " -f1 | tr '[:upper:]' '[:lower:]')
```

Comment:
The username from git config does not necessarily as the same as the gitlab repo name. In my case, I have to change my user name in git config to execute the `./configure`.

Is it necessary to grab the username from git config instead of command line input?

Reply:
Hi @kencho51,

Your course of action was a bit radical ;-)
You could have simply edited the` .env` file and adjusted the value of REPO_NAME, CI_PROJECT_URL and PROJECT_VARIABLES_URL to match your fork name.

In any case,  I agree that Gitlab fork name cannot be guessed reliably.

However, command line based configuration has to aim to facilitate automation (that was the initial motivation)
so interactive inputs with the user like we have for the Terraform init script is not good at all, and we should not replicate that.

So I have now changed the initial config file creation so that:

by default, `REPO_NAME` behaves like `GITLAB_PRIVATE_TOKEN`: 
It's initially populated with "replace-me" and `makeDotSecrets()` will output an error inviting user to edit .env to correct the value of REPO_NAME (and there is **no** need to change CI_PROJECT_URL and PROJECT_VARIABLES_URL)


You needed to edit the `.env` file anyway to update the private token, so it's not adding new steps to the workflow, and gives you opportunity to review `.env`'s content.

Reply:
Hi @rija,

Thank you for the update,  updating the `REPO_NAME` in .env together with `GITLAB_PRIVATE_TOKEN` is much simpler now.